files/jira.rb

```diff
@@ -52,6 +52,10 @@ def create_issue(summary, description, project)
           issue_json['fields'].merge!({'components' => component.map { |i| {'name' => i} }})
         end
 
+        if handler_settings['priority_map'].include?(priority)
```

Comment:
Does this need to be

```if handler_settings['priority'] && handler_settings['priority_map'].include?(priority)```

Or in other words, will priority handler_settings always be there, even as empty hash?

Reply:
I believe it should be, since that's the [default it's passed in as](https://github.com/Yelp/sensu_handlers/pull/130/files#diff-60ae41fd0a31977447947f59940ee9a4R58), and that's passed along [through to the handler](https://github.com/Yelp/sensu_handlers/pull/130/files#diff-e78a34e94a5fe8f6121ca380e58696a0R31). The main danger here I think is that we misconfigure it so that it's no longer a hash and it becomes something that doesn't respond to `include?` any more, but I think that's unlikely to happen in practice.